src/spectral.ts

```diff
@@ -14,11 +18,49 @@ function configureRuleset(parser: Parser): RulesetDefinition {
   return {
     extends: [aasRuleset],
     rules: {
-      'asyncapi-schemas-v2': aas2schemaParserRule(parser),
+      'asyncapi-is-asyncapi': asyncApi2IsAsyncAPI(),
+      'asyncapi-schemas-v2': aas2SchemaParserRule(parser),
       // We do not use these rules from the official ruleset due to the fact 
       // that the given rules validate only AsyncAPI Schemas and prevent defining schemas in other formats 
       'asyncapi-payload-unsupported-schemaFormat': 'off',
       'asyncapi-payload': 'off',
     },
   } as RulesetDefinition;
 }
+
+function asyncApi2IsAsyncAPI(): RuleDefinition {
```

Comment:
```suggestion
function asyncApi2IsAsyncApi(): RuleDefinition {
```

Or the other way around, but to unify namings

Reply:
Thanks!